### DISCUSSION TITLE
deep learning on tabular data with text columns

### DISCUSSION THREAD

**Discussion by wjlgatech** [manujosephv/pytorch_tabular#45] [Q&A]
Hi Joseph, Great job to make this amazing package! I am doing research to use deep learning techniques to model tabular data with text columns. Do you happen to know any existing techniques out there? Do you have plan to add this feature into your package? I am working to make it happen using fastai2 existing modules. If you are interested, we can meet up to chat more. Thank you!

Paul

**Comment by manujosephv** [comment]
@wjlgatech That's a good use case, but something that is not supported out if the box yet by the library... PyTorch wide-and-deep is a very similar library but has more focus on multi modal problems like the one you have now.. Probably you can check that out..

**Reply by wjlgatech** [reply]
@manujosephv thanks for the reference! I am checking it out.
